src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -585,12 +585,43 @@
 examples:
   - name: Create role assignment for an assignee.
     text: az role assignment create --assignee sp_name --role a_role
+  - name: Create role assignment for an assignee with description and condition. (Only for 2020-04-01-preview API version and later.)
+    text: >-
```

Comment:
What's the difference between `>-` and `|`?

Reply:
@qwordy 
![image](https://user-images.githubusercontent.com/3250203/93058781-66ea7e00-f6a2-11ea-9920-b9229116e5f1.png)
In short: `>-` will replace all single `\n` to whitespace, and combine double `\n` to one.

Reply:
I use `>-` to make the code more readable purposefully.